values.yaml

```
@@ -3,6 +3,10 @@
 
 # Available parameters and their default values for the Vault chart.
 
+# Allows to specify namespace other than Release.Namespace
+# If empty, default namespace would be Release.Namespace
+namespaceOverride: ""
```

WDYT about just `namespace`? It feels descriptive enough to me.

```suggestion
namespace: ""
```